app/components/ThirdwebProvider.tsx

```diff
@@ -0,0 +1,3 @@
+"use client"
```

Comment:
By remomving 'use client' from layout.tsx we ensure the ssr functioning in general, unless otherwise specified in specific components like here.